ext/dom/tests/DOMXPath_quote.phpt

```diff
@@ -0,0 +1,84 @@
+--TEST--
+Test DOMXPath::quote with various inputs
+--SKIPIF--
+<?php if (!class_exists('DOMXPath')) die('skip DOMXPath not available.'); ?>
+--FILE--
```

Comment:
Missing an Extension section

Reply:
Thanks

Btw, it's not important, but could you go to https://github.com/php/php-src/pull/13456/files# and hold down the left mouse button on line 3 and move the mouse pointer to line 4 and then release the left mouse button, and then make a comment with the content
````
```suggestion
--EXTENSIONS--
dom
```
````
? (if you think this procedure sounds unnecessarily complex, i think so too and suggested to simplify it here https://github.com/orgs/community/discussions/109469 )

Reply:
nvm, it's fixed :)